docs/getting-started/installing-substrate.md

```diff
@@ -2,37 +2,44 @@
 title: "Installing Substrate"
 ---
 
+Following ways to install Substrate assume you are running Mac OS, Arch, or a Debian-based OS like Ubuntu. For Windows users, you can try it [here](https://github.com/paritytech/substrate#612-windows).
+
+## Fast Installation
+
+If you plan to create your own blockchain, then run this simple one-liner:
+
+```bash
+curl https://getsubstrate.io -sSf | bash -s -- --fast
+```
+
+This command will install the [substrate-up](https://github.com/paritytech/substrate-up) commands for creating your own nodes and modules.
+
+This is the most common way to start your Substrate's journey.
+
 ## Full Installation
 
-If you're running Mac OS, Arch, or a Debian-based OS like Ubuntu, then **with a good internet connection** run this simple one-liner:
+If you want to experience the latest Substrate's features and **have a good internet connection**, then run command:
 
 ```bash
 curl https://getsubstrate.io -sSf | bash
 ```
 
-This command will download and compile the complete substrate node, and the subkey utility and install them on your path. It will also install the substrate-up scripts for quickly creating your own nodes and modules.
+This command will:
+* Install latest [subkey](https://github.com/paritytech/substrate/tree/master/subkey) which generate or restore the keys used for Substrate chains.
+* Install latest Substrate [node](https://github.com/paritytech/substrate/tree/master/node) for you to have a taste of all the available Substrate's features.
+* Install the [substrate-up](https://github.com/paritytech/substrate-up) scripts for quickly creating your own nodes and modules.
 
-It will take 20 - 40 minutes to finish. Once it is done, check that Substrate is properly installed by running `substrate --version`. You should see something like this:
+It will take 20 ~ 40 minutes to finish, depends on your hardware and network connection. Once it is done, check that Substrate is properly installed by running `substrate --version`. You should see something like this:
```

Comment:
```suggestion
It will take 20 to 40 minutes to finish, depending on your hardware and network connection. Once it is done, check that Substrate is properly installed by running `substrate --version`. You should see something like this:
```